com.librumreader.librum.yaml

```diff
@@ -0,0 +1,37 @@
+app-id: com.librumreader.librum
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: librum
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=home
+  - --device=dri
+modules:
+  - python3-setuptools_scm.yaml
+  - python3-scikit-build.yaml
+  - python3-swig.yaml
+  - python3-libclang.yaml
+  - name: librum
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release 
+      - -DBUILD_TESTS=Off
+    sources:
+        - type: git
+          url: https://github.com/Librum-Reader/Librum.git
+          commit: 13386c7185f37e1f9525f86ba959d5fcf1e0f7ef
+  - name: install-files
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 librum.desktop -t /app/share/applications
+      - install -Dm644 com.librumreader.librum.metainfo.xml -t /app/share/metainfo
+      - install -Dm644 src/logo.svg -t /app/share/icons/hicolor/scalable/apps
```

Comment:
this is not the icon name specified in the .desktop

Reply:
Right, the icon name in my .desktop file is `librum.svg`. Am I expected to change this, to whatever the `rename-icon` operation will call it, or can I simply change the name of the logo to `librum.svg` and leave the rest as it is?

Reply:
it's `librum` in the .desktop file. which is **consistent**. `rename-icon` will fix the desktop file which is why I suggested it.

I both case the icon name **exclude** the extension because you can also have bitmap icons allong the svg.

Reply:
> I both case the icon name exclude the extension because you can also have bitmap icons allong the svg.

I am not sure if I understand this. Am I supposed to remove the .svg only for the `rename-icon` command?

I have just made a commit about it. Is that everything that I needed to do, or is there still something wrong with the icon?